src/Device.vala

```diff
@@ -0,0 +1,75 @@
+public class Sound.Device : GLib.Object {
```

Comment:
Missing header?

Reply:
@jeremypw Will add. Should I include elementary LLC in the copyright header, or just myself and @DropsOfSerenity ?

Reply:
I would just copy the header from `PulseAudioManager.vala`, change the date to "2021" and substitute yourself as an author

Reply:
Added the missing copyright headers. Thanks for the feedback!